packaging/windows/install_script.nsis

```diff
@@ -0,0 +1,237 @@
+Unicode true
+# This installs two files, agent-windows-amd64.exe and logo.ico, creates a start menu shortcut, builds an uninstaller, and
+# adds uninstall information to the registry for Add/Remove Programs
+  
+!define APPNAME "Grafana Agent"
+!define DESCRIPTION "The Grafana Cloud Agent collects observability data and sends it to a compatible remote write endpoint"
+# These will be displayed by the "Click here for support information" link in "Add/Remove Programs"
+!define HELPURL "https://github.com/grafana/agent" # "Support Information" link
+!define UPDATEURL "https://github.com/grafana/agent" # "Product Updates" link
```

Comment:
Can help URL point to the discussions page, and update URL point to the releases page?